scripts/README.md

```diff
@@ -0,0 +1,156 @@
+## Script directory
+This directory contains scripts that can be used to generate CYCLUS
+input files and analyze CYCLUS output files.
+
+### analysis.py
+
+Input : CYCLUS output file (.sqlite)  
+```
+python analysis.py [outputfile]
+```
+
+Most functions return a dictionary of lists (timeseries of a value)
+that can be used to plot a stacked bar chart or a line plot.
+
+### create_input.py
+Python script to create CYCLUS input files from PRIS Year-end Reator Status 
+Reports. Once the Report is downloaded from the PRIS Database, it must be 
+saved as ``database/Year-end Reactor Status_####.csv``, in which the ``####``
+is the four digit year the data is pulled from. Creates multiple files that 
+are used for a CYCLUS simulation, using functions contained in 
+``predicting_the_past_import.py``. Due to relative path dependancies in this 
+script, it **must** be run from the ``scripts`` directory.
+
+Input : None
+
+User Specifications : 
+- l. 7 : ``data_year`` (int) -- four digit year the data is pulled from, must match 
```

Comment:
Consider removing the `1.` on these bullet points.

Reply:
That is an `l`, not a 1. As in `line 7`. I will expand that out since it is not clear.